src/actionlist/action-list-tree.scss

```diff
@@ -0,0 +1,97 @@
+// <ul>
+// tree-view variant
+// renders ActionList with default styling for tree-view
+// collapse/expand icons presents as leadingActions, rotate 90deg
+// connecting vertical lines between collapse groups
+// consistent font-size between nested groups
+
+@mixin treeConnectingLine($left) {
```

Comment:
@vdepizzol can you help me debug what is happening here? 🙈 its so close.. but overflowing for some reason

![image](https://user-images.githubusercontent.com/18661030/145920513-d24b4991-79fb-4ce6-b042-dcc7aec14540.png)